src/locales/vocabulary.js

```diff
@@ -99,47 +100,42 @@ export default {
   'editdialog.anonymousMessage2_login': 'log in to OpenStreetMap',
   'editdialog.anonymousMessage3': ', your changes will be immediate.',
   'editdialog.add_major_tag': 'Add',
-  'editdialog.location_checkbox': 'Change location',
+  'editdialog.location_checkbox': 'New location',
   'editdialog.location_placeholder': 'eg. across the street',
   'editdialog.location_editor_to_be_added': 'The position cannot be edited here yet, you can do so in the <a href="__link__">iD editor</a>.',
-  'editdialog.place_cancelled': 'Place permanently closed (delete)',
+  'editdialog.place_cancelled': 'Permanently closed (delete)',
   'editdialog.comment': 'Comment (optional)',
-  'editdialog.comment_placeholder': 'link to the source of information etc.',
+  'editdialog.comment_placeholder': 'Extra details, link to source, etc.',
   'editdialog.info_edit': `Your edit will be immediately saved to the OpenStreetMap. Please,
          enter only information from your own or verified sources. It is prohibited
          to copy copyrighted data (e.g. Google Maps). <a href="https://wiki.openstreetmap.org/wiki/How_We_Map">More info</a>`,
-  'editdialog.info_note': `Your suggestion will be processed by OpenStreetMap volunteers. Here
-         you can add an additional note or describe position adjustment etc.
-         It is also appropriate to support your contribution with a link to a source of information (web,
-         photo, etc.).`,
-  'editdialog.other_tags': 'Other properties – tags',
+  'editdialog.info_note': `Your suggestion will be reviewed by editors. You can add additional information for them below! If possible, please provide a link to the source that your suggestion is based on.`,
```

Comment:
I prefer `OpenStreetMap volunteers` than `editors`. The point of osmapp was to educate users on how it works..

Why omitting `(web, photo, etc.)`? I think its always better to add examples, don't you think?

Reply:
I chose to omit because they are not exclusive. Since you can't post files in the box, only text, in order to submit an image it would have to be a web link anyway. So either way you're posting a web link.

"Web" is also a bit unnatural and doesn't specify anything.

Maybe:
> Your suggestion will be reviewed by OpenStreetMap volunteers. You can add additional information such as a link to a photo or a link to source material for them below!

Reply:
Yess! Thats much better! Thanks 🙂